smithy-model/src/test/resources/software/amazon/smithy/model/loader/invalid/properties-v2-only.smithy

```diff
@@ -0,0 +1,13 @@
+//[ERROR] example.weather#City: Parse error at line 10, column 2 near `\n`: Resource properties can only be used with Smithy version 2 or later. Attempted to use resource properties with version `1.0`. | Model
```

Comment:
I think  this just uses substring matching. To make this test more resiient, try just proving the string "Resource properties can only be used with Smithy version 2 or later" followed by " | Model"

```suggestion
// Resource properties can only be used with Smithy version 2 or later | Model
```

Reply:
The suggestion itself doesn't work. The trailing `| Model` isn't the substring.